plugins/Files/js/components/usagestats.js

```diff
@@ -1,15 +1,40 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 
-const UsageStats = ({allowance, spending}) => (
-	<div className="files-usage-info">
-		<span> {spending} SC Spent / {allowance} SC Allocated </span>
-	</div>
-)
+// UsageStats defines the presentation component for displaying file spending.
+const UsageStats = ({allowance, downloadspending, uploadspending, storagespending, contractspending, renewheight}) => {
+	const unspent = () => allowance - (downloadspending+uploadspending+storagespending+contractspending)
```

Comment:
why is this a function? Did it used to take an argument?

Reply:
I think so, not needed anymore